app/models/sales_log.rb

```diff
@@ -42,14 +42,32 @@ class SalesLog < Log
   }
   scope :filter_by_organisation, ->(owning_organisation, _user = nil) { where(owning_organisation:) }
   scope :filter_by_owning_organisation, ->(owning_organisation, _user = nil) { where(owning_organisation:) }
+  scope :age1_answered, -> { where.not(age1: nil).or(where(age1_known: [1, 2])) }
```

Comment:
Similar point to the lettings log, is it avoidable having hard coded specific questions answered tied to the model? I think it's about levels of abstraction and that a sales log model shouldn't be concerned with specific questions and if they're answered.